qrencode.el

```diff
@@ -1084,6 +1084,15 @@ Commands:
         (message "No URL found at point")
       (qrencode--encode-to-buffer url))))
 
+;;;###autoload
+(defun qrencode-url (url)
+  "Encode URL.
+When called interacively, read URL with `read-string'."
+  (interactive `(,(read-string "URL: ")))
```

Comment:
Why the macro use and read-string? Wouldn't `(interactive "sURL: ")` achieve the same?

Reply:
Thanks -- I've been writing a lot of stuff recently with weird interactive inputs -- forgot about the string codes. Fixed.